src/treecodes.py

```diff
@@ -213,15 +221,15 @@ def getGroup(name):
 
 def getGroupCode(name):
     try:
-        code = tbplnames[name]['group']
+        code = tbplnames[name.strip()]['group']
```

Comment:
some tests' name have useless blank (e.g ' mochitest-devtools-chrome-e10s-4') and it make we can't get the group symbol.